src/components/v2/Toggle/index.tsx

```diff
@@ -1,19 +1,23 @@
+/** @jsxImportSource @emotion/react */
 import React from 'react';
 import Switch from '@mui/material/Switch';
+import { SerializedStyles } from '@emotion/react';
 import { useStyles } from './styles';
 
 export interface IToggleProps {
   onChange: (event: React.ChangeEvent<HTMLInputElement>, checked: boolean) => void;
   value: boolean;
+  cssStyles?: SerializedStyles;
```

Comment:
```suggestion
  className?: string;
```

You'll then need to apply the `className` prop to the `Switch` component tag.

It is counterintuitive, but passing the `className` means components can pass a `css` prop directly to `Toggle` and it will be applied to it. You can check the `Tabs` component (`src/components/v2/Tabs/index.tsx`) to see an example.

Reply:
I think either approach is valid, but we should be consistent. Currently we have been passing the className and not the css object

Reply:
I think either approach is valid, but we should be consistent. Currently we have been passing the className and not the css object